src/DynamicFormControl.vue

```diff
@@ -43,6 +47,20 @@
             formControl: Object,
             colWidth: String
         },
+        methods: {
+            isRed(value){
+                if (value){
+                    if(value.constructor === Array){
+                        if (value.length > 0){
+                            return false
+                        } else return true
+                    }
+                }
+                if(value){
```

Comment:
lines 59 - 61 can be simplified to `return !!value`

Reply:
That's a good point. Hint doesn't have any booleans as it uses Yes/No dropdowns instead. I'm not sure about mint.

Reply:
Might not be possible to get a boolean here, but for the avoidance of unexpected behaviour now or in the future probably a good idea to check explicitly for undefined values! 0 is also falsy so need to be careful.